cmd/svcat/testdata/responses/clusterservicebrokers/ups-broker.json

```diff
@@ -24,8 +24,8 @@
         "type": "Ready",
         "status": "True",
         "lastTransitionTime": "2018-01-11T20:53:31Z",
-        "reason": "FetchedCatalog",
-        "message": "Successfully fetched catalog entries from broker."
+        "reason": "FetchedClusterCatalog",
+        "message": "Successfully fetched cluster catalog entries from broker."
```

Comment:
Same thing with 'cluster' here - I don't think the additional wording is going to make a difference to users beyond confusing them.